PublishParasoftResults/task.json

```diff
@@ -60,6 +60,14 @@
             "required": false,
             "helpMarkDown": "Relative location in project check out directory or absolute location for settings file to load rule documentation.\n \\- configure \"dtp.url\" property to specify DTP server address."
         },
+        {
+          "name": "referenceBuild",
+          "type": "string",
+          "label": "Reference build",
+          "defaultValue": "",
+          "required": false,
+          "helpMarkDown": "Specify a reference build number. For example, if you want to specify a reference build which run name is '#20230927.1 • Update azure-pipelines.yml for Azure Pipelines', the build number here should be '20230927.1'. To use the last successful build, leave this field empty.\n This field allows users to filter the results by baseline state in the Scans UI and also applies to the quality checks."
```

Comment:
Use "Specify a reference build number which can be found in pipeline run title after #, e.g. build number is 20 in title '#20 • commit message'. To use the last successful build, leave this field empty. It will be used as a reference build if Parasoft Quality Gates task is configured."